packages/flutter/lib/src/cupertino/localizations.dart

```diff
@@ -92,6 +95,7 @@ abstract class CupertinoLocalizations {
   ///
   /// - US English: Wed Sep 27
   /// - Russian: ср сент. 27
+  // Based on intl package's DateFormat.MMMEd.
```

Comment:
Just based on the other similar comments ... The global version is based on ...?

Reply:
True. Changed